docs/content/docs.md

```diff
@@ -1066,7 +1066,7 @@ Mode to run dedupe command in.  One of `interactive`, `skip`, `first`,
 `newest`, `oldest`, `rename`.  The default is `interactive`.  
 See the dedupe command for more information as to what these options mean.
 
-### --default-time TIME
+### --default-time Time
```

Comment:
Add an `=` in this one?

Reply:
Not sure what you mean... I've tried to make headers consistently use `--option value` instead of `--option=value` for now, so this follows that. Do you mean we should swap all of these? Or is there something special about this specific one?

Reply:
The other changes in this commit all use `=` but I think you change them all later!